src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -313,44 +313,33 @@ private object CoerceText(object baseValue)
 #endif
 		object Description
 		{
-			get => (object)this.GetValue(DescriptionProperty);
-			set
-			{
-				this.SetValue(DescriptionProperty, value);
-			}
+			get => this.GetValue(DescriptionProperty);
+			set => this.SetValue(DescriptionProperty, value);
 		}
 
 		public static DependencyProperty DescriptionProperty { get; } =
 			DependencyProperty.Register(
-				"Description",
+				nameof(Description),
 				typeof(object),
 				typeof(TextBox),
 				new FrameworkPropertyMetadata(
 					defaultValue: null,
-					propertyChangedCallback: (s, e) => ((TextBox)s)?.OnDescriptionChanged(e)
+					propertyChangedCallback: (s, e) => ((TextBox)s)?.UpdateDescriptionVisibility(false)
 				)
 			);
 
-		private void OnDescriptionChanged(DependencyPropertyChangedEventArgs args)
+		private void UpdateDescriptionVisibility(bool initialization)
 		{
-			ContentPresenter descriptionPresenter = this.FindName("DescriptionPresenter") as ContentPresenter;
+			if (initialization && Description == null)
```

Comment:
Just personally curious what purpose the `initialization` argument serves. I'm guessing it serves as a way to defer loading the control until it makes sense. Why wouldn't `Description == null` alone be sufficient then? Other than my question, this new pattern is probably superior and definitely more concise.

Reply:
@duke7553 yes, as Jerome mentioned, that is the reason - to avoid eagerly loading the element unless explicitly required. It is just minor change, but could have performance benefits 🙂